classes/class-amp-admanager.php

```diff
@@ -430,4 +449,35 @@ public function load_amp_resources() {
 
 		}
 	}
+
+	/**
+	 * Add amp sticky ad to the footer.
+	 *
+	 * @return void
+	 */
+	public function add_sticky_amp_ad() {
+
+		if ( empty( self::$amp_settings['amp_admanager_enable_sticky_ads'] ) || '1' !== self::$amp_settings['amp_admanager_enable_sticky_ads'] ) {
+			return;
+		}
+
+		if ( empty( self::$amp_settings['amp_admanager_sticky_ad_unit'] ) ) {
+			return;
+		}
+
+		?>
+		<script async custom-element="amp-sticky-ad" src="https://cdn.ampproject.org/v0/amp-sticky-ad-1.0.js"></script> <?php // phpcs:ignore WordPress.WP.EnqueuedResources.NonEnqueuedScript ?>
+
+		<?php
+
+		$ad_attr = [
+			'ad-unit'       => self::$amp_settings['amp_admanager_sticky_ad_unit'],
+			'desktop-sizes' => '600x90,728x90',
+			'tablet-sizes'  => '320x50,468x60',
+			'mobile-sizes'  => '320x50,468x60',
```

Comment:
sizes should be dynamic instead of static, right? I see the discussion is going on #59 
cc: @pradeep910

Reply:
Yes sizes should be dynamic, and override with these `*-sizes` parameters.